client/wfm/src/task/TaskRepository.ts

```diff
@@ -0,0 +1,5 @@
+import {Task} from './Task';
+export interface TaskRepository {
+  getAll(): Promise<Task[]>;
```

Comment:
Probably, doesn't make sense to grab *all* Tasks in the world, not using it anywhere in the example atm, but just thought maybe all repositories would have CRUD**L**